src/packages/application/initialize.js

```diff
@@ -103,7 +104,8 @@ export default async function initialize<T: Application>(app: T, {
 
   const server = new Server({
```

Comment:
I think it would be beneficial to spread the `serverOpts` here:

```javascript
  const server = new Server({
    ...serverOpts,
    router,
    logger,
  });
```